contracts/PermissionedNodeRegistry.sol

```diff
@@ -390,19 +391,49 @@ contract PermissionedNodeRegistry is
     }
 
     /**
-     * @notice update the name and reward address of an operator
+     * @notice propose the new reward address of an operator
+     * @dev only the existing reward address (msg.sender) can propose
+     * @param _operatorAddress operator address
+     * @param _rewardAddress new reward address
+     */
+    function initiateRewardAddressChange(address _operatorAddress, address _rewardAddress) external override {
+        UtilLib.checkNonZeroAddress(_rewardAddress);
+        uint256 _operatorId = operatorIDByAddress[_operatorAddress];
```

Comment:
pass operator ID instead of _operatorAddress to skip line 400 and 401

Reply:
vivek asked to pass operator address